CorePluginFramework/test/unit/src/au/gov/asd/tac/constellation/plugins/gui/MultiChoiceInputPaneNGTest.java

```diff
@@ -0,0 +1,106 @@
+
+    private static final String PLANETS_PARAMETER_ID
+            = PluginParameter.buildId(MultiChoiceInputPaneNGTest.class, "planets");
+    private final PluginParameter<MultiChoiceParameterType.MultiChoiceParameterValue> planetOptions = MultiChoiceParameterType.build(PLANETS_PARAMETER_ID);
+    final PluginParameters params = new PluginParameters();
+
+    public MultiChoiceInputPaneNGTest() {
+    }
+
+    @BeforeClass
+    public static void setUpClass() throws Exception {
+        if (!FxToolkit.isFXApplicationThreadRunning()) {
+            FxToolkit.registerPrimaryStage();
+        }
+    }
+
+    @AfterClass
+    public static void tearDownClass() throws Exception {
+        try {
+            FxToolkit.cleanupStages();
+        } catch (TimeoutException ex) {
+            LOGGER.log(Level.WARNING, "FxToolkit timedout trying to cleanup stages", ex);
+        }
+    }
+
+    @BeforeMethod
+    public void setUpMethod() throws Exception {
+        if (!planetOptions.getName().equals("Planets")) {
+            planetOptions.setName("Planets");
+            planetOptions.setDescription("Some planets");
+            planets = Arrays.asList("Mercury", "Venus", "Earth", "Mars", "Jupiter", "Saturn", "Uranus", "Neptune", "Coruscant");
+            MultiChoiceParameterType.setOptions(planetOptions, planets);
+            final List<String> checked = new ArrayList<>();
+            checked.add("Earth");
+            MultiChoiceParameterType.setChoices(planetOptions, checked);
+            params.addParameter(planetOptions);
+        }
+    }
+    private List<String> planets;
+
+
+    @Test
+    public void multiChoiceInputPane_constructorWithOptions() {
+        MultiChoiceInputPane multiChoiceInputPane = spy(
+                new MultiChoiceInputPane((PluginParameter<MultiChoiceParameterType.MultiChoiceParameterValue>) params.getParameters().get(PLANETS_PARAMETER_ID)));
+        assertTrue(multiChoiceInputPane.input.getAllMenuItems().size() == planets.size());
+        assertTrue(multiChoiceInputPane.getFieldValue().size() == planetOptions.getMultiChoiceValue().getChoicesData().size());
+        assertTrue(multiChoiceInputPane.getFieldValue().getFirst().toString().equals(planetOptions.getMultiChoiceValue().getChoicesData().getFirst().toString()));
+    }
+    
+    @Test
+    public void multiChoiceInputPane_getFieldListener() {
+        MultiChoiceInputPane multiChoiceInputPane = spy(
+                new MultiChoiceInputPane((PluginParameter<MultiChoiceParameterType.MultiChoiceParameterValue>) params.getParameters().get(PLANETS_PARAMETER_ID)));
+        
+        ConstellationInputListener<List<ParameterValue>> fieldChangeListenerMock = mock(ConstellationInputListener.class);
+        doReturn(fieldChangeListenerMock).when(multiChoiceInputPane).getFieldChangeListener(planetOptions);        
+        ConstellationInputListener<List<ParameterValue>> fieldChangeListener = multiChoiceInputPane.getFieldChangeListener(planetOptions);        
```

Comment:
`fieldChangeListener` is never used ? 
is the next line supposed to be using this ?